feat: migrate Markdown lint to `@eslint/markdown` (#5593)
mochajs/mocha: docs/index.md

description: "Mocha is a feature-rich JavaScript test framework running on Node.js and in the browser, making asynchronous testing simple and fun."
---

> <!-- Disable markdown/no-missing-link-fragments due to the rule not aware of this file is specifically converted to HTML -->

Reviewer: ```suggestion
<!-- False positive as this file is specifically converted to HTML -->
```
